rotkehlchen/tests/unit/decoders/test_curve.py

```diff
@@ -61,6 +62,26 @@
     from rotkehlchen.types import ChecksumEvmAddress
 
 
+def _populate_arb_eure_cache() -> None:
```

Comment:
perhaps make it a fixture and set the scope to "module" so it only gets run once. then, use it in the tests

Reply:
It can't be module because it depends in the globaldb that is function scope. The globaldb gets destroyed and re-created on every test